src/picomsg/src/oper_db_writer.h

```diff
@@ -20,6 +21,8 @@ bool oper_db_writer_set_terminator_tunnel_items(const char *vrf_name, const char
 bool oper_db_writer_set_terminator_tunnel_oper_data(const struct terminator_tunnel_oper_data *oper_data, bool write_uptime);
 bool oper_db_writer_del_terminator_tunnel_oper_data(const char *vrf_name, const char *device_id);
 bool oper_db_writer_set_terminator_tunnel_oper_data_ike_spis(const char *vrf_name, const char *device_id, enum ike_sa_state_t status, uint64_t inbound_ike_spi, uint64_t outbound_ike_spi);
+bool oper_db_writer_copy_global_intervals(const char *vrf_name, const char *device_id, struct terminator_tunnel_lifetime_intervals intervals);
+bool oper_db_writer_set_uptime(const char *vrf_name, const char *device_id, enum LIBOPER_DB_CLIENT_FIELD_IDS field_id);
```

Comment:
Don't accept a `field_id` parameter, it is too confusing and error-prone.
Instead split this function to `oper_db_writer_set_tunnel_ike_sa_uptime()` and `oper_db_writer_set_tunnel_ike_sa_auth_uptime()`.
Internally in `oper_db_writer.cpp` it's fine they'll be implemented using a helper function such as this, which accepts a  `LIBOPER_DB_CLIENT_FIELD_IDS`